src/bandwidth-tool/templates/app.vue

```diff
@@ -80,6 +90,8 @@ limitations under the License.
     const dropletData = require('../../build/droplets');
     const droplets = {};
     for (const droplet of dropletData) {
+        if (!droplet.available) continue;
```

Comment:
This could be `if (!droplet.available || !droplet.regions.length) continue;`

Reply:
Good call.